plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemoryConnectorTest.java

```diff
@@ -92,6 +92,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                  SUPPORTS_ADD_FIELD,
                  SUPPORTS_AGGREGATION_PUSHDOWN,
                  SUPPORTS_CREATE_MATERIALIZED_VIEW,
+                 SUPPORTS_DEFAULT_COLUMN_VALUE,
```

Comment:
Memory connector could support default column value right ? It would also allow us to validate the code path - where the default value is added by the engine today

Reply:
Right. However, the connector doesn't support MERGE statement.